### PR TITLE
Charmis/sdk upgradation

### PR DESCRIPTION
**Pull request by CharmisG**
plugin compileSDK and targetSDk Upgradation to 31

### CODE REVIEW

File: android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -16,5 +16,19 @@ limitations under the License.
 
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="com.squareup.sdk.reader.react">
+          
+    <uses-permission
+        android:name="android.permission.BLUETOOTH"
```
Comment (hukid) [inline android/src/main/AndroidManifest.xml:21]: Why do we need permission for this file?

Comment (CharmisG) [reply android/src/main/AndroidManifest.xml:21]: For support above android 12 versions.

File: reader-sdk-react-native-quickstart/ios/File.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
+//
+//  File.swift
+//  RNReaderSDKSample
+//
+//  Created by Bharat Cholera on 01/07/22.
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/ios/File.swift:5]: please update this header comment.

Comment (hukid) [reply reader-sdk-react-native-quickstart/ios/File.swift:5]: - [ ] update the header comment.

File: reader-sdk-react-native-quickstart/ios/File.swift [hunk 3]
```diff
@@ -0,0 +1,32 @@
+//
+//  File.swift
```
Comment (hukid) [inline reader-sdk-react-native-quickstart/ios/File.swift:2]: Can we have a better name for this file?

- [ ] use a better file name.

**Review by hukid — APPROVED** [review]

**Pull request merged**